samples/Demo Data/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
 
 // Android Resolver Exclusions Start
 android {
+	namespace "com.unity3d.player"
```

Comment:
```suggestion
  namespace "com.unity3d.player"
```

should this value be like `com.datadog.unity.xxx`?

Reply:
This got inserted by Unity.... I'll double check it it's actually a setting somewhere that I didn't override properly.